triggers/tests/BUILD.bazel

```diff
@@ -108,16 +106,23 @@ client_server_test(
     server_files = ["$(rootpath :acs.dar)"],
 )
 
-da_scala_test_suite(
+da_scala_test(
```

Comment:
They depend on each other so `scala_test_suite` doesn’t work. Also they run in less than 15 seconds on my machine so it doesn’t seem that useful to parallelize them.